src/templates/aids/detail.html

```diff
@@ -4,6 +4,16 @@
 
 {% block extratitle %}{{ aid.name }}{% endblock %}
 
+{% block extra_head %}
+<meta property="og:title" content="{{ aid.name }}" />
+<meta property="og:type" content="website" />
+<meta property="og:description" content="Trouvez les aides pertinentes pour financer et accompagner les projets de votre territoire" />
+<meta property="og:url" content="{{ request.build_absolute_uri }}" />
+<meta property="og:site_name" content="Aides-territoires" />
+<meta property="og:image" content="https://aides-territoires.beta.gouv.fr/static/img/logo_AT_og.png" />
```

Comment:
```suggestion
<meta property="og:image" content="/static/img/logo_AT_og.png" />
```
non ? (idem pour le home.html)

Reply:
Je crois que non https://indieweb.org/The-Open-Graph-protocol#Disallows_relative_URLs :)